### PR TITLE
Enhance expire to continue processing when a repo fails.

### PR DESCRIPTION
**Pull request by cmwshang**
When the expire command fails on a single repo it will continue on to the next one rather than stopping.

Errors are logged to the log file rather than thrown. If, after processing all repos, one or more errors occurred, then it will throw a single error to indicate there were errors and the log file should be inspected.

### CODE REVIEW

File: src/command/expire/expire.c [hunk 1]
```diff
@@ -900,68 +899,103 @@ cmdExpire(void)
...
+            CATCH_ANY()
+            {
+                LOG_ERROR_FMT(
+                    errorTypeCode(errorType()), "repo%u: %s", cfgOptionGroupIdxToKey(cfgOptGrpRepo, repoIdx), errorMessage());
+                repoError = true;
+                errorTotal++;
+            }
+            TRY_END();
+
+            if (!repoError)
```
Comment (dwsteele) [inline src/command/expire/expire.c:968]: I can't figure out why expire is broken into two try blocks. Why not just run everything in the first try block?

File: src/command/expire/expire.c [hunk 2]
```diff
@@ -900,68 +899,103 @@ cmdExpire(void)
                 THROW_FMT(OptionInvalidValueError, "'%s' is not a valid backup label format", strZ(adhocBackupLabel));
         }
 
+        // Track any errors that may occur
+        unsigned int errorTotal = 0;
```
Comment (dwsteele) [inline src/command/expire/expire.c:903]: Seems like a simple boolean would do the job.

**Review by dwsteele — CHANGES_REQUESTED** [review]

File: src/command/expire/expire.c (hunk 1 above)
Comment (cmwshang) [reply src/command/expire/expire.c]: Fixed https://github.com/pgbackrest/pgbackrest/pull/1308/commits/54e3bf08da92f1c157dc6d3f79f80439e4f35265

File: src/command/expire/expire.c (hunk 2 above)
Comment (cmwshang) [reply src/command/expire/expire.c:903]: Kept as count and added it to final THROW. Commit https://github.com/pgbackrest/pgbackrest/pull/1308/commits/54e3bf08da92f1c157dc6d3f79f80439e4f35265

**Review by dwsteele — APPROVED** [review]

**Pull request merged**